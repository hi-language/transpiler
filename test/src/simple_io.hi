// This test should pass with the current transpiler.

message: "Hello from Hi!"
_(message)

count: 1
count = 2
_("Count is", count)
