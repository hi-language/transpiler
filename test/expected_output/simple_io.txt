Hello from Hi!
Count is 2
